selectors-4/Overview.bs

```diff
@@ -2256,20 +2256,9 @@ The Focus-Indicated Pseudo-class: '':focus-visible''</h3>
 		like clicking on the button with a mouse pointer.
 	</div>
 
-	<div class=advisement>
-		Page authors should follow these guidelines
-		when deciding whether to use '':focus'' or '':focus-visible''
-		to style the focused state of an element:
-
-		* If the element has “native” focus indicator behavior
-			(such as text fields or buttons),
-			use '':focus-visible''.
-		* Otherwise, if the element is emulating a text input,
-			or something else that is <em>intended</em> to receive keyboard interaction,
-			use '':focus''.
-		* Otherwise,
-			use '':focus-visible''.
-	</div>
+	This pseudo-class is intended to give developers the ability to style focus
+	selectively, in order to encourage visible, usable focus styles, and
+	discourage the practice of disabling the default user agent focus style.
 
 	When UAs choose to specially indicate focus on an element,
```

Comment:
I think this paragraph confuses the issue about the problem we're trying to solve. The problem is the practice of disabling the default UA focus style, mentioned in the previous paragraph.